### PR TITLE
add config json for compatibility with sequelize-cli.

### PR DESCRIPTION
**Pull request by bakavic**
PR #13 didn't fix the issue correctly - cli couldn't read the config.

I've changed how the config is stored - it differs from the old style of storing the config in `sequelize.js`.

### CODE REVIEW

File: config/sequelize.js [hunk 1]
```diff
@@ -1,43 +1,47 @@
-exports.default = { 
+var sequelizeConfig = require('./sequelize.json');
```
Comment (evantahler) [inline config/sequelize.js]: No, I don't want to change the config for this plugin to `.json`.  The point of having all config as `.js` files is that it allows more logic (reading from ENV, etc).

**Comment by bakavic** [comment]
Hmm, I have modified `sequelize.js` to be usable by sequelize-cli. Are the new changes acceptable?

Not too sure how I should emulate the AH style of config priority in a cleaner fashion - any suggestions?

**Comment by evantahler** [comment]
This looks OK!

**Comment by evantahler** [comment]
```
> npm publish
git push --tags
+ ah-sequelize-plugin@0.5.0
```

**Pull request merged**